includes/agentic-commerce/class-wc-stripe-agentic-commerce-inventory-tracker.php

```
@@ -103,6 +103,10 @@ public function register_hooks(): void {
 	 * @return void
 	 */
 	public function track_stock_change( \WC_Product $product ): void {
+		if ( ! WC_Stripe_Agentic_Commerce_Product_Mapper::should_sync_product( $product ) ) {
```

PHPMD (2.15.0)

[error] Avoid using static access to class 'WC_Stripe_Agentic_Commerce_Product_Mapper' in method 'track_stock_change'. (undefined)

(StaticAccess)